samples/.gitignore

```diff
@@ -0,0 +1 @@
+*/Properties
```

Comment:
Running the samples caused them to create property folders, which showed up in git status. That kind of thing drives me bonkers, so I hid them.